src/rpc/util.cpp

```diff
@@ -66,3 +66,55 @@ CScript CreateMultisigRedeemscript(const int required, const std::vector<CPubKey
 
     return result;
 }
+
+UniValue DescribeAddressVisitor::operator()(const CNoDestination &dest) const {
+    return UniValue(UniValue::VOBJ);
+}
+
+UniValue DescribeAddressVisitor::operator()(const CKeyID &keyID) const {
+    UniValue obj(UniValue::VOBJ);
+    CPubKey vchPubKey;
```

Comment:
In https://github.com/bitcoin/bitcoin/pull/10583/commits/4dadd7c7ac0934447b8ca33942535cc8a22e78f5

Unused variable?